### ISSUE TITLE
Non-English charset

### ISSUE THREAD

**Issue by AzamRabiee** [keithito/tacotron#26]
@keithito Thanks for the nice and clean code!
I've tried your code for a non-English dataset. I did not mess up the code. Also, in hyperparameters, only sample_rate is tuned. The loss is decreasing well. The ground truth generated wav files are also great. But the alignment plots are meaningless from the scratch. Here you can find some examples
[alignment-plots.zip](https://github.com/keithito/tacotron/files/1240805/alignment-plots.zip)
Do you have any idea?

**Comment by keithito** [comment]
Hi, what language are you training on? The input pipeline converts everything to ASCII, so it will have trouble if your language doesn't use latin characters:
https://github.com/keithito/tacotron/blob/master/util/textinput.py#L37

Depending on the language, you may want to change the character set used in textinput.py or transliterate your input into latin characters. A few people have had success training on Chinese data. They might be able to share what they did to get things to work.

**Comment by AzamRabiee** [comment]
oh~ 
I see!
So, do you have any idea for Korean characters?

**Comment by keithito** [comment]
Yes, I think that's the right solution. You can also remove _uppercase and _lowercase from _valid_input_chars if they don't appear in your dataset.

I'm glad you were able to figure this out. The code in textinput.py is pretty confusing and I've been meaning to clean it up when I have some time.

**Comment by AzamRabiee** [comment]
Your code is great!
I got some promising output in just one day! at 74k gs!
